### ISSUE TITLE
Jakarta compatible version of ews-java-api

### ISSUE THREAD

**Issue by paolobazzi** [OfficeDev/ews-java-api#773]
The ews-java-api library depends on Java EE javax-packages in two classes:

- microsoft.exchange.webservices.data.core.request.ServiceRequestBase
- microsoft.exchange.webservices.data.property.complex.TimeChange

This makes it impossible to use this library on a Java EE Jakarta-based application using version >=9, see https://jakarta.ee/specifications/platform/9/.

We migrated the ews-java-api as local patch for our application.
If there is any interest to provide a Jakarta compatible version of ews-java-api I could provide a pull request with this changes.

**Comment by davidmoten** [comment]
You do realise the last commit here was 5 years ago?

**Comment by paolobazzi** [comment]
Yes of course, that was the reason for this question instead of directly creating a pull request.
As we have a working patch for our application we're fine. I would only create a pull request if there is any interest by other users.

**Comment by ivanchong123** [comment]
> The ews-java-api library depends on Java EE javax-packages in two classes:
> 
> * microsoft.exchange.webservices.data.core.request.ServiceRequestBase
> * microsoft.exchange.webservices.data.property.complex.TimeChange
> 
> This makes it impossible to use this library on a Java EE Jakarta-based application using version >=9, see https://jakarta.ee/specifications/platform/9/.
> 
> We migrated the ews-java-api as local patch for our application. If there is any interest to provide a Jakarta compatible version of ews-java-api I could provide a pull request with this changes.

Hi, i am interest on the Jakarta compatible version of ews-java-api. Can you please provide the pull request of the changes?

**Comment by nateko** [comment]
> Yes of course, that was the reason for this question instead of directly creating a pull request. As we have a working patch for our application we're fine. I would only create a pull request if there is any interest by other users.

Please create a pull request when you can. Many people are interested.

**Comment by paolobazzi** [comment]
I added a simple patch - basically just two imports need to be adjusted and the according dependencies needs to be udpated to a jakarta-compatible version.
